monai/inferers/utils.py

```diff
@@ -167,8 +169,8 @@ def sliding_window_inference(
     importance_map_ = convert_data_type(importance_map_, torch.Tensor, device, compute_dtype)[0]
 
     # handle non-positive weights
-    min_non_zero = max(importance_map_[importance_map_ != 0].min().item(), 1e-3)
-    importance_map_ = torch.clamp(importance_map_.to(torch.float32), min=min_non_zero).to(compute_dtype)
+    min_non_zero = max(torch.min(importance_map_).item(), 1e-3)
```

Comment:
I can't remember why there is a `importance_map_[importance_map_ != 0]` cc @Nic-Ma 
removing the logic doesn't raise any test error in this PR so I assume it's ok to remove.

Reply:
Hi @wyli ,

Adil added this feature in PR: https://github.com/Project-MONAI/MONAI/pull/355 before.
The comment for this line code is: "importance_map cannot be 0, otherwise we may end up with nans!".

Thanks.

Reply:
I think with the current `max` and `clamp`, the map will always be positive. now the difference is to use the smallest positive element or 1e-3 for the zero and negative weights. is there a case that the difference becomes important?

Reply:
the `clamping` logic does not change. and `importance_map_` would remain positive after changes. I am not sure why 1e-3 is chosen.